vignettes/baselinenowcast.Rmd

```diff
@@ -194,7 +198,7 @@ reporting triangle.
 # observed long above.
 observed_long <- data_long[location == "DE"][age_group == "00+"] |> # nolint
   enw_filter_report_dates(latest_date = nowcast_date) |>
-  enw_filter_reference_dates(include_days = n_history_delay - 1)
+  enw_filter_reference_dates(include_days = n_history_delay + n_history_uncertainty - 1) # nolint
```

Comment:
you compute this at least twice. I suggest you save it as a variable and reuse it